src/css/simple-mde.css

```diff
@@ -0,0 +1,750 @@
+/**
+ * simplemde v1.11.2
+ * Copyright Next Step Webs, Inc.
+ * @link https://github.com/NextStepWebs/simplemde-markdown-editor
+ * @license MIT
+ */
```

Comment:
Ideally we’d like to avoid ever having to vendor dependencies into source control—package managers are of course much easier to work with. In this case I think we should just be able to add `'node_modules/simplemde/dist/simplemde.min.css'` to [this list](https://github.com/popcodeorg/popcode/blob/master/gulpfile.js#L77-L80). We’ll also want to add `simplemde` itself as a dependency in `package.json` so we can guarantee that it will be hoisted to the top level, but that seems fine.